### ISSUE TITLE
Questions about the training.

### ISSUE THREAD

**Issue by YunjiePeng** [tteepe/GaitGraph#15]
As written in the './experiments/1_train_casia-b.sh', the 'train_data_path' and the 'valid_data_path' are set to '../data/casia-b_pose_train_valid.csv' and '../data/casia-b_pose_test.csv', respectively. It seems that the division of the training set (74 subjects) conducted in the data processing, which separates the training set (74) into a training set (59) and a validation set (15), is not used during training. Instead, the testing set (50 subjects) is employed as the validation set to pick the best model.

No offense, but this sounds not reasonable and seems unfair to compare with other skeleton-based gait recognition methods. Under the settings of the training set (59), the validation set (15), and the testing set (50)，the results obtained by me are lower than those written in the paper: NM84.02, CL69.99, BG60.08.

**Comment by srikamalteja** [comment]
@YunjiePeng Actually I think, test set is further divided into Gallery set and Probe subsets.

The results achieved by this paper is on Probe set (#NM 5-6, #CL - 1-2, #BG 1-2)

Please go through the `train.py` and `evaluate.py` scripts and feel free to correct me if I'm wrong

**Comment by tteepe** [comment]
Since there are not official splits, we use the splits commonly used for CASIA-B. Please consult the paper for the details on the splits.
